tekton/verify-tekton-pipeline-cosign-signated-images/verify-tekton-pipelines-cosign-signed-images.yml

```diff
@@ -0,0 +1,17 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: require-signed-images-registry
+spec:
+  validationFailureAction: Enforce
+  background: true
+  rules:
+    - name: check-signature
+      match:
+        any:
+          - resources:
+              kinds:
+                - Pod
```

Comment:
This is simply verifying a `Pod` and not verifying a `TaskRun`. What is the intent?

Reply:
Intent is Restrict creation of TaskRun pod resources if images are not signed using Cosign
  This is part of Tekton Chains.
All images build via tekton should be signed images part of supply chain security